resources/assets/css/_buttons.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .button-primary:hover {
-    @apply bg-theme-primary-700 shadow-button-primary;
```

Comment:
can also remove the styling from the tailwind config file if we get rid of it for the buttons

Reply:
there's also other parts that use `shadow-button-primary`, e.g. the markdown editor styling to mimic the regular buttons for modals